src/coreclr/vm/wasm/cgencpu.h

```diff
@@ -32,7 +32,8 @@ struct HijackArgs
 
 inline LPVOID STDCALL GetCurrentSP()
 {
-    _ASSERTE("The function is not implemented on wasm, it lacks registers");
+    // WASM-TODO: check where we reach that, return nullptr for now to not break runtime initialization
+    //PORTABILITY_ASSERT("The function is not implemented on wasm, it lacks registers");
```

Comment:
How are we getting here? Can you share a callstack? I would prefer us to nicely ifdef out the caller in an intelligent way.